lib/transactional.pm

```diff
@@ -66,16 +67,18 @@ sub process_reboot {
     my (%args) = @_;
     $args{trigger}            //= 0;
     $args{automated_rollback} //= 0;
+    $args{expected_grub}      //= 1;
 
     handle_first_grub if ($args{automated_rollback});
 
     if (is_microos) {
         microos_reboot $args{trigger};
     } else {
         power_action('reboot', observe => !$args{trigger}, keepconsole => 1);
-        if (check_var('ARCH', 's390x')) {
+        if (check_var('ARCH', 's390x') || check_var('BACKEND', 'pvm_hmc')) {
```

Comment:
Don't you mind to change all of these `check_var('BACKEND', 'pvm_hmc')` calls with `is_pvm` from `lib/Utils/Backends.pm`? I know that we are inconsistent with that, but if we don't introduce new code with `check_var` we have a chance to get rid of those calls and use documented wrapper methods.

Reply:
good idea, fixed.